server/game/player.js

```diff
@@ -483,6 +483,8 @@ class Player extends Spectator {
             // as it will cause then to double their effects when attached later.
             let isSetupAttachment = playingType === 'setup' && card.getType() === 'attachment';
 
+            var originalLocation = card.location;
```

Comment:
`let` instead of `var` (normally not too picky about this but there's a `let` on the previous line)